app-frontend/src/app/pages/projects/detail/detail.controller.js

```diff
@@ -121,6 +127,26 @@ export default class ProjectsDetailController {
         });
     }
 
+    populateExportList(page) {
+        if (this.loadingExports) {
+            return;
+        }
+        delete this.exportLoadingError;
+        this.loadingExports = true;
+        this.exportList = [];
+        this.projectService.listExports({
+            sort: 'createdAt, desc',
+            pageSize: 10,
+            page: page - 1,
```

Comment:
this should be `page > 0 ? page - 1 : 1` otherwise you can end up with a page = -1 if the page param is not set 
Same for scenes